package.json

```diff
@@ -796,15 +787,6 @@
                 "excludePath": [],
                 "excludepackages": "com.azure.messaging.servicebus.implementation"
             },
-            {
```

Comment:
I see that you're removing these from CI. Are we removing them entirely?

Reply:
I am not quite sure about this. I proposed to remove it in the issue listed in description. Java team does not review it yet. These packages does not in the mono repo at all. I am thinking to remove them from doc as well

Reply:
>These packages do not in the mono repo at all.  

what do you mean by this? Are you saying that this documentation isn't current in docs? I disagree.

![image](https://user-images.githubusercontent.com/45376673/95519843-d2123200-097a-11eb-9e4d-a40855609c94.png)

https://review.docs.microsoft.com/en-us/java/api/overview/azure/core?view=azure-java-preview&branch=master

Reply:
Make certain you check the `preview` moniker!

Reply:
Resolved after discussion offline. `tracing-opentelemetry` is retired, as is `tracing-opencensus`.